client/src/app/s-degree-plan/s-degree-plan.component.ts

```diff
@@ -14,16 +14,25 @@ import { share } from "rxjs/operators";
 })
 export class SDegreePlanComponent implements OnInit {
   yearArray: Observable<Array<Year>>;
+  openPanel = false;
 
   constructor(
     private router: Router,
     private errorHandler: ErrorHandler,
     private planService: PlanService
   ) {
     this.yearArray = this.planService.formatPlan();
+    this.openPanel = true;
     this.yearArray.subscribe((result) => console.log(result));
```

Comment:
Is this subscription necessary? It only seems to log the result.

Reply:
No it is not, this subscription was already there. I can delete it